server/notifier.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/applikatoni/applikatoni/models"
+)
+
+func generateSummary(t *template.Template, a *models.Application, d *models.Deployment, u *models.User, success bool) (string, error) {
```

Comment:
I think it would be neater to remove the `success bool` here and instead check the state on the deployment (which we're loading from DB here anyway): https://github.com/applikatoni/applikatoni/blob/376a190efff67563648fb958874faae2f7a14949/models/deployment.go#L7-L18